dockernet/config.sh

```diff
@@ -354,17 +357,21 @@ WAIT_FOR_STRING() {
   ( tail -f -n0 $1 & ) | grep -q "$2"
 }
 
+# Sleep until the balance has changed
+# Optionally provide a minimum amount it must change by (to ignore interest)
 WAIT_FOR_BALANCE_CHANGE() {
   chain=$1
   address=$2
   denom=$3
+  minimum_change=${4:-1} # defaults to 1
 
   max_blocks=30
 
   main_cmd=$(GET_VAR_VALUE ${chain}_MAIN_CMD)
-  initial_balance=$($main_cmd q bank balances $address --denom $denom | grep amount)
+  initial_balance=$($main_cmd q bank balances $address --denom $denom | grep amount | NUMBERS_ONLY)
   for i in $(seq $max_blocks); do
-    new_balance=$($main_cmd q bank balances $address --denom $denom | grep amount)
+    new_balance=$($main_cmd q bank balances $address --denom $denom | grep amount | NUMBERS_ONLY)
+    balance_change=$(echo "$new_balance - $initial_balance" | bc)
 
     if [[ "$new_balance" != "$initial_balance" ]]; then
```

Comment:
Doesn't seem like minimum_change defined on 366 or balance_change on 374 are being used in this func?  Should this line be something like "$balance_change" >= "$minimum_change"

Reply:
Similar comment for the next function Wait for delegation change as well

Reply:
Actually looks like `minimum_change` is defined as a param in all these fns but not used in any of the fns

Reply:
good catch! updated